packages/grpc-js/src/channel.ts

```diff
@@ -230,6 +239,14 @@ export class ChannelImplementation implements Channel {
       new MaxMessageSizeFilterFactory(this.options),
       new CompressionFilterFactory(this),
     ]);
+
+    this.callRefTimer = setInterval(() => {}, 1 << 31 - 1);
```

Comment:
Isn't there a maxint constant?

Reply:
There is `Number.MAX_SAFE_INTEGER`, but it's larger than that, and `Number.MAX_VALUE`, which is even larger.